packages/modules/web_themes/standard/source/src/stores/mqtt-store.ts

```diff
@@ -1553,17 +1567,68 @@ export const useMqttStore = defineStore('mqtt', () => {
       : [];
   };
 
+  /**
+   * Toggle the active state of the scheduled charging plan identified by the scheduled charge plan id
```

Comment:
Die Methode ist nicht auf das Umschalten (toggle) beschränkt, sondern übernimmt den neuen Wert aus dem Aufruf.
```suggestion
   * Get or set the active state of the scheduled charging plan identified by the scheduled charge plan id
```

Reply:
Changed